### PR TITLE
Conmon image update

### PR DESCRIPTION
**Pull request by haircommander**
carries: https://github.com/containers/conmon/pull/152/files
plus a patch to force the installation of cri-o to address https://github.com/containers/conmon/pull/152#issuecomment-629477014

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: contrib/cirrus/setup_environment.sh [hunk 1]
```diff
@@ -54,31 +55,44 @@ then
...
+			# to build_and_replace_conmon below
+            ooe.sh rpm --force -ivh $PACKAGE_DOWNLOAD_DIR/*.rpm
+
             install_testing_deps
             build_and_replace_conmon
+            build_and_replace_bats
 
             cd "$CRIO_SRC"  # cri-o source
             echo "Building binaries required for testing"
+			ooe.sh make vendor
```
Comment (haircommander) [inline contrib/cirrus/setup_environment.sh:86]: nit, whitespace

**Comment by rhatdan** [comment]
@haircommander Still working on this one?

**Comment by haircommander** [comment]
> @haircommander Still working on this one?

it is on the list yes. I may need more guidance from @cevich

**Comment by cevich** [comment]
Okay, I'm back now.  @haircommander in which direction would you like me to point my finger or eyeballs?

**Comment by haircommander** [comment]
welcome back! Back to the world of fires for you :smiley: 

It seems the bats suite isn't actually running. do you know how to get access to these vms to debug? I haven't figured out why they're not running

**Comment by cevich** [comment]
In the `integration` output I see:
```
...
install  -D -m 755 bin/conmon /usr/bin/conmon
...
```

The Makefile in this repo is missing the definition.  In libpod, it's done like this:

```
SELINUXOPT ?= $(shell test -x /usr/sbin/selinuxenabled && selinuxenabled && echo -Z)
```

The error however, is coming from the `integration_test.sh` call `GOCACHE=/tmp/go-build make vendor`.  Is this make target known to be functional otherwise?

**Comment by haircommander** [comment]
> In the `integration` output I see:
> 
> ```
> ...
> install  -D -m 755 bin/conmon /usr/bin/conmon
> ...
> ```
> 
> The Makefile in this repo is missing the definition. In libpod, it's done like this:
> 
> ```
> SELINUXOPT ?= $(shell test -x /usr/sbin/selinuxenabled && selinuxenabled && echo -Z)
> ```


the selinux enabled bool seems to be a build flag for podman itself. conmon does not use it, so it shouldn't be necessary

> The error however, is coming from the `integration_test.sh` call `GOCACHE=/tmp/go-build make vendor`. Is this make target known to be functional otherwise?

what's the point of using GOCACHE here? the target is functional otherwise

**Comment by cevich** [comment]
> the selinux enabled bool seems to be a build flag for podman itself. conmon does not use it, so it shouldn't be necessary

Naw, it's an `install` (command) flag.  It makes sure the correct context is set on the file after it's put in place.

> what's the point of using GOCACHE here? the target is functional otherwise

I was questioning that as well.  I think it was added as a workaround for some obscure go tooling bug.  I'm also wondering about the wisdom of even using `make vendor` at this level.  I'm going to try splitting that out into a separate task...

**Comment by cevich** [comment]
I've rebased my PR from this one, and am continuing to poke at the `integration` task failures...

**Comment by rhatdan** [comment]
@cevich Any progress?

**Comment by rhatdan** [comment]
This is so old, I am closing. I have updated the cirrus image in my man page PR.

**Pull request closed without merging**